demo/api/src/warnings/entities/warning.entity.ts

```diff
@@ -17,11 +17,11 @@ export class Warning extends BaseEntity {
     @Field(() => ID)
     id: string = uuid();
 
-    @Property()
+    @Property({ columnType: "timestamp with time zone" })
```

Comment:
Please use type instead of columnType, see https://github.com/vivid-planet/comet-starter/pull/581.

Reply:
Maybe this isn't possible because of this:

> Additionally I changed createdAt and updatedAt to timestamp with time zone. Before it was timestamptz(0) which had no millisecond precision. This lead to deleting some warnings that were created in the same second.

Reply:
I changed it to type, and run `npm run mikro-orm migration:create`

And it did not create migrations for the warning table

Reply:
I believe this change removes the the filter and sort options for the GraphQL API, which are probably needed. So should we revert it?